server/services/twilio/twilio.main.js

```diff
@@ -8,18 +8,19 @@ function send( to, from, body, attempts ) {
         to: '+1' + to,
         from: '+1' + from,
         body: body
-    }, function( err, data ){
-        if( err ) {
+    }, function( error, data ){
+        if (error) {
+            log.error('Sending Monthly Options text using twilio falied', {error}, {to, from, message: body, resendAttempts: attempts} );
```

Comment:
@mLuby in the last object the `to` and `from` vars are stings which will be turned into keys but they have no value assigned... (e.g. `, key: value,` vs `, key,`) wont this break?